native/src/main/java/io/ballerina/stdlib/ai/wso2/Native.java

```diff
@@ -38,6 +44,10 @@
  * @since 1.0.0
  */
 public class Native {
+    public static final String ANY_OF = "anyOf";
+    public static final String BALLERINA_AI = "ballerina/ai:";
```

Comment:
imo, we should exclude the `:` in the constant variable.
Ok for do in a separate PR